### PR TITLE
Deploy to fargate

### PR DESCRIPTION
**Pull request by chaspy**
ref: https://github.com/chaspy/ask/issues/10

### CODE REVIEW

**Comment by chaspy** [comment]
Deploy success ✅ at 294ba30
https://circleci.com/gh/chaspy/ask/57?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: .circleci/config.yml [hunk 1]
```diff
@@ -28,9 +28,34 @@ jobs:
...
+          command: |
+            sudo pip install -r requirements.txt
+      - save_cache:
+          key: v1-{{ checksum "requirements.txt" }}
+          paths:
+            - "venv"
+      - run: |
+          curl https://raw.githubusercontent.com/silinternational/ecs-deploy/master/ecs-deploy | sudo tee /usr/local/bin/ecs-deploy
+          sudo chmod +x /usr/local/bin/ecs-deploy
+      - run: ecs-deploy -n ask-service -k $AWS_ACCESS_KEY_ID -s $AWS_SECRET_ACCESS_KEY -r $AWS_DEFAULT_REGION -c ask-cluster -i chaspy/ask:latest
```
Comment (chaspy) [inline .circleci/config.yml]: I've added the environment variables to CircleCI
https://circleci.com/gh/chaspy/ask/edit#env-vars
screenshot 2019-02-09 22 33 19

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+awscli==1.16.101
```
Comment (chaspy) [inline requirements.txt]: I installed awscli in container, and made this file.
ref: 294ba30

**Comment by ariririri** [comment]
@chaspy sorry, I can't undestand what you did.
Thus. please tell me how I confirm your result.

**Comment by chaspy** [comment]
caching is wrong~
revert to WIP.

**Comment by chaspy** [comment]
Deploy successful ✅ at 11203cd
CI: https://circleci.com/gh/chaspy/ask/75?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: .circleci/config.yml [hunk 3]
```diff
@@ -28,9 +28,40 @@ jobs:
...
               docker push chaspy/ask:latest
             fi
+  deploy:
+    docker:
+      - image: circleci/python:3.6.1
+    environment:
+      AWS_DEFAULT_OUTPUT: json
+    steps:
+      - checkout
+      - restore_cache:
```
Comment (chaspy) [inline .circleci/config.yml]: package caching
ref: https://circleci.com/docs/2.0/caching/

File: .circleci/config.yml [hunk 4]
```diff
@@ -28,9 +28,40 @@ jobs:
...
+      - save_cache:
+          key: v1-{{ checksum "requirements.txt" }}
+          paths:
+            - venv
+      - run: |
+          curl https://raw.githubusercontent.com/silinternational/ecs-deploy/master/ecs-deploy | sudo tee /usr/local/bin/ecs-deploy
+          sudo chmod +x /usr/local/bin/ecs-deploy
+      - run: |
+          . venv/bin/activate
+          ecs-deploy -n ask-service -k $AWS_ACCESS_KEY_ID -s $AWS_SECRET_ACCESS_KEY -r $AWS_DEFAULT_REGION -c ask-cluster -i chaspy/ask:latest -t 180
```
Comment (chaspy) [inline .circleci/config.yml]: I've added the environment variables to CircleCI
https://circleci.com/gh/chaspy/ask/edit#env-vars
screenshot 2019-02-09 22 33 19

**Comment by chaspy** [comment]
@ariririri 
Please review and merge

**Comment by ariririri** [comment]
@chaspy LGTM, I'll merge it

**Pull request merged**